_about/notes/index.md

```diff
@@ -15,8 +15,8 @@ toc: false
 
 {% include section-index.html docs=site.docs %}
 
-
-- The [latest](https://github.com/istio/istio/releases) Istio monthly release is {{site.data.istio.version}}. It is downloaded when the following is used(*):
+- The [latest](https://github.com/istio/istio/releases) Istio monthly release is {{site.data.istio.version}}.
```

Comment:
note that this worked better when site.data.istio.version was an actual version, aka 0.5.1 vs 0.6preliminary

also change that file ? to put 0.6 or 0.6.0 if we don't want to update it for each patch